VectorRumble.Shared/Helper.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.Xna.Framework;
+
+namespace VectorRumble
+{
+    internal static class Helper
+    {
+        const string MyDocumentsDirectory = "/Documents"; // Making this a constant in case we need to IFDEF this per platform later
+
+        public static string GetMyDocumentsFolder ()
+        {
+            // Once our app is signed, we can't change it, so let's look in the Special "MyDocuments" folder for user created data
+            var myDocuments = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
+
+            if (!myDocuments.ToLower().Contains("documents")) {
+                myDocuments += MyDocumentsDirectory;
```

Comment:
Can we not use a Path.Combine here instead? That will take care of the platform specific path seperators.

Reply:
Good point! Not thinking.